CHANGELOG_PENDING.md

```diff
@@ -44,6 +44,9 @@
 - [sdk/nodejs] Allow prompt values in `construct` for multi-lang components.
   [#6522](https://github.com/pulumi/pulumi/pull/6522)
 
+- [automation/dotnet] Fix (de)serialization of StackSettings.
```

Comment:
Would this be considered breaking? It will not automatically read users's files in the previous serialization? 

Ah I recall automation/dotnet is pre-GA so this may be ok.

Reply:
I this this was just broken prior to this fix because pulumi wouldn't be able to use the incorrectly cased keys.

This should be under Bug Fixes though. Might be worth just lumping all of nodejs/python/dotnet under one changelog entry.

Reply:
I'll merge this PR, rebase the other PR I have open, and consolidate all the entries into one.